src/norm/postgres.nim

```diff
@@ -217,7 +217,7 @@ proc insert*[T: Model](dbConn; objs: var openArray[T], force = false) =
   for obj in objs.mitems:
     dbConn.insert(obj, force)
 
-proc select*[T: Model](dbConn; obj: var T, cond: string, params: varargs[DbValue, dbValue]) {.raises: {NotFoundError, ValueError, DbError, LoggingError}.} =
+proc select*[T: Model](dbConn; obj: var T, cond: string, params: varargs[DbValue, dbValue]) {.raises: {NotFoundError, ValueError, DbError, LoggingError, Exception}.} =
```

Comment:
What is the point of this generic exception? Doesn't it include all possible exceptions?

Reply:
Nim appears to have gotten a damn sight tighter about its exception tracking system.
You'll notice that `log` also raises Exception. That's because the procs `log` calls (`logLoop` from logging.nim) can raise `Exception`. That in turn means that `select`, which calls `log`, can raise `Exception`. Therefore, in order for this to compile, we **must** add `Exception` there. Same for all the other places as well as `log.nim`.

Reply:
I don't think it is the right way. You might somehow remove `Exception` from the raises lists of callees.

Reply:
For example, you might put `log` under the `try` statements

```nim
try:
  log ...
except Exception:
  raises newexception(LoggingError, ...)
```